doc/admin-guide/files/records.yaml.en.rst

```diff
@@ -4785,6 +4785,17 @@ removed in the future without prior notice.
 
    Disables HTTP/0.9 over QUIC by default.
 
+.. ts:cv:: CONFIG proxy.config.quic.quiche_cc_algorithm INT 0
```

Comment:
I don't think we should have "quiche" in the setting name. We wouldn't want "quic.openssl_cc_althorithm" in addition to this.